### DISCUSSION TITLE
Set widths for slides in freeMode?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [nolimits4web/swiper#4267] [Q&A] [answered]
Hi,

Is it possible to have slides at a set with (with set spacing between them) in freeMode?  This would be as opposed to having to specify the number of slides per view.

An example would be like [this](https://stayinguncomfortable.com/) where the slide items can be fixed and exist partially out of the screen.

Thanks!

**Comment by nolimits4web** [comment, marked as answer]
Yes, set `slidesPerView: 'auto'` and set slide's size in CSS, for example:
```css
.swiper-slide {
  width: 320px;
}
```

**Reply by dungle-scrubs** [reply]
Thanks, works great
